search-app/docs/api.md

```diff
@@ -210,6 +210,13 @@ See the [CMR Client Partner User Guide](https://wiki.earthdata.nasa.gov/display/
     * [Reset the application to the initial state](#reset-the-application-to-the-initial-state)
     * [Querying caches](#querying-caches)
     * [Check application health](#check-application-health)
+  * [Associate Concepts](#generic-association)
+    * [Associate Multiple Concepts](#generic-multi-association)
+    * [Associate Concepts By Revision](#generic-association-with-revisions)
+    * [Associate Concepts With Details](#generic-association-with-details)
+    * [Search For Associations](#generic-association-search)
+    * [Disassociate Concepts](#generic-association-delete)
+    * [Disassociate Multiple Concepts](#generic-association-multi-delete)
```

Comment:
I don't think it's necessary to have a section for multiple association and dissociations.  People are already familiar with a concept being associated/dissociated with multiple concepts. If you want to associate with only one concept, that's just special case for the multiple concepts. So, you just need to cover the multiple case, without calling it multiple. If the user wants to only associate one, it's their choice.